packages/components/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -42,9 +43,9 @@ export function Breadcrumbs({
 
   return (
     <nav aria-label="breadcrumbs" role="navigation">
-      <RACBreadcrumbs items={itemsWithRoot || items}>
+      <RACBreadcrumbs className={className} items={itemsWithRoot || items}>
         {(item) => (
-          <RACBreadcrumb id={item['@id']}>
+          <RACBreadcrumb className={className} id={item['@id']}>
```

Comment:
@Gomez not sure to use the same class in both, this could lead to style leaks. :/